test/16.js

```diff
@@ -0,0 +1,12 @@
+const kebabCase = require('../solutions/16.js');
+const test = (string, result) => {
+  if (kebabCase(string) === result) {
+    console.log('Yay it works.');
```

Comment:
Can we change the `console.log` to print out the actual answer? We want to let user know what the function is returning.